teuthology/task/print.py

```diff
@@ -1,5 +1,16 @@
 """
 Print task
+
+A task that simply prints the output that is given to it as
+an argument. Can be used like any other task (under sequential,
+etc...)
```

Comment:
If we're going to add this, how about making it crystal clear that it actually logs the output.  "Print" means "terminal" to me, and of course that makes no sense for scheduled jobs, but one might think this is not useful for scheduled jobs (and it is).  In fact, I'm not sure why it's not called "log"